Signal/src/AppDelegate.m

```diff
@@ -248,6 +248,8 @@ - (void)setupEnvironment
 
     self.staleNotificationObserver = [OWSStaleNotificationObserver new];
     [self.staleNotificationObserver startObserving];
+
+    [TSPreKeyManager sharedManager];
```

Comment:
Since we're using this line of code (a getter) only for it's side effects, I think we need a comment.

But at that point, it might be more concise to move the `observation` out of the init, and just say `[[TSPreKeyManager sharedManager] observeNotifications]` which speaks for itself.